internal/pkg/manifest/workload_ecs.go

```diff
@@ -91,17 +91,33 @@ type AlarmArgs struct {
 	MemoryUtilization *float64 `yaml:"memory_utilization"`
 }
 
+// WorkerAlarmArgs represents specs of CloudWatch alarms for Worker Service deployment rollbacks.
+type WorkerAlarmArgs struct {
+	AlarmArgs
+	MessagesDelayed *int `yaml:"messages_delayed"`
+}
+
 // DeploymentConfiguration represents the deployment strategies for a service.
 type DeploymentConfiguration struct {
 	Rolling        *string                    `yaml:"rolling"`
-	RollbackAlarms Union[[]string, AlarmArgs] // `yaml:"rollback_alarms"` 
+	RollbackAlarms Union[[]string, AlarmArgs] // `yaml:"rollback_alarms"`
 	// The rollback_alarms manifest field is a no-op until the EDS-CFN ABR bug is fixed.
 }
 
+// WorkerDeploymentConfig represents the deployment strategies for a worker service.
+type WorkerDeploymentConfig struct {
+	Rolling              *string                          `yaml:"rolling"`
+	WorkerRollbackAlarms Union[[]string, WorkerAlarmArgs] `yaml:"rollback_alarms"`
```

Comment:
should we comment this?
```suggestion
	WorkerRollbackAlarms Union[[]string, WorkerAlarmArgs] // `yaml:"rollback_alarms"`
```